src/internal/call.c

```diff
@@ -0,0 +1,76 @@
+#include <rlang.h>
+
+#include "internal.h"
+
+
+static bool is_callable(sexp* x) {
+  switch (r_typeof(x)) {
+  case r_type_symbol:
+  case r_type_call:
+  case r_type_closure:
+  case r_type_builtin:
+  case r_type_special:
+    return true;
+  default:
+    return false;
+  }
+}
+
+static sexp* rlang_call2(sexp* fn, sexp* args, sexp* ns) {
+  if (r_typeof(fn) == r_type_character) {
+    if (r_length(fn) != 1) {
+      r_abort("`.fn` must be a length 1 string");
```

Comment:
Can we just say "must be a string"? (i.e. a string is a length 1 character vector)